source/client-side-encryption/tests/README.rst

```diff
@@ -128,15 +128,16 @@ Then for each element in ``tests``:
    #. Insert the data specified into the ``admin.datakeys`` with write concern "majority".
 
 #. Create a MongoClient.
-   
+
 #. Create a collection object from the MongoClient, using the ``database_name``
-   and ``collection_name`` fields from the YAML file. If a ``json_schema`` is defined in the test,
+   and ``collection_name`` fields from the YAML file. Drop the collection 
+   with writeConcern "majority". If a ``json_schema`` is defined in the test,
    use the ``createCollection`` command to explicitly create the collection:
+
```

Comment:
Drive-by fix. These instructions were saying to drop the collection after creating it with a JSON schema.